browser/main/NoteList/index.js

```diff
@@ -574,22 +574,29 @@ class NoteList extends React.Component {
     })
   }
 
+  confirmDeleteNote (selectedNotes) {
```

Comment:
This function is duplicated of [it](https://github.com/BoostIO/Boostnote/blob/master/browser/main/Detail/index.js#L36).
I want you to extract to `browser/lib`.